pkg/network/node/vnids.go

```diff
@@ -136,6 +136,18 @@ func (vmap *nodeVNIDMap) getVNID(name string) (uint32, error) {
 	return 0, fmt.Errorf("failed to find netid for namespace: %s in vnid map", name)
 }
 
+func (vmap *nodeVNIDMap) findDuplicateNetID(namespace string, netID uint32) (bool) {
+	names := vmap.GetNamespaces(netID)
+	for _, name := range names {
+		if name != namespace {
+			klog.Warningf("Netid %d for namespace %s already exists under different namespace %s",
+						  netID, namespace, name)
```

Comment:
indentation is wrong here, which is making the `verify` test fail. Run `gofmt` to fix it